.gitignore

```diff
@@ -21,7 +21,8 @@ gcs_key_file.json
 *_build/
 *_unity/
 *_unity_seperate/
-*_unity_separate/
+output/
```

Comment:
The other separate was intentional (different spellings, with the a one being the correct one)